dev.pulsar_edit.Pulsar.metainfo.xml

```diff
@@ -3,21 +3,26 @@
   <id>dev.pulsar_edit.Pulsar</id>
   <name>Pulsar</name>
   <developer_name>Pulsar Edit</developer_name>
-  <summary>A Community-led Hyper-Hackable Text Editor</summary>
+  <summary>Community-led hyper-hackable editor</summary>
   <launchable type="desktop-id">dev.pulsar_edit.Pulsar.desktop</launchable>
   <metadata_license>MIT</metadata_license>
   <project_license>MIT</project_license>
   <url type="homepage">https://pulsar-edit.dev</url>
   <url type="bugtracker">https://github.com/pulsar-edit/pulsar/issues/</url>
   <url type="faq">https://pulsar-edit.dev/docs/launch-manual/sections/faq/</url>
   <url type="vcs-browser">https://github.com/pulsar-edit/pulsar</url>
+  <branding>
+  <color type="primary" scheme_preference="light">#c061cb</color>
+  <color type="primary" scheme_preference="dark">#613583</color>
+  </branding>
   <update_contact>admin@pulsar-edit.dev</update_contact>
   <description>
     <p>Pulsar aims to not only reach feature parity with the original Atom, but to bring Pulsar into the 21st century by updating the underlying architecture, and supporting modern features.</p>
   </description>
   <screenshots>
     <screenshot type="default">
-      <image type="source">https://raw.githubusercontent.com/pulsar-edit/pulsar/master/resources/readme.png</image>
+      <image xml:lang="en" type="source">https://raw.githubusercontent.com/pulsar-edit/pulsar/master/resources/readme.png</image>
+      <caption xml:lang="en">Pulsar Edit welcome window with an open project</caption>
```

Comment:
See the note https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#metainfo-translations